### PR TITLE
feat: add list of webhooks to forum, stage, and voice channels

### LINKED ISSUE #1176: Get webhooks in Forum, Stage, and Voice channels
### Summary

Add a function to get a list of webhooks for forum, stage, and voice channels

### What is the feature request for?

The core library

### The Problem

Currently can't get a list of webhooks in these types of channels

### The Ideal Solution

Add a function called webhooks to ForumChannel and VoiceGuildChannel(or StageChannel and VoiceChannel) classes to get a list of webhooks

### The Current Solution

Set ForumChannel.webhooks, StageChannel.webhooks, and VoiceChannel.webhooks equal to TextChannel.webhooks

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by acco4321**
## Summary
Adds .webhooks to ForumChannels, StageChannels, and VoiceChannels.

Resolves #1176 

## This is a **Code Change**

- [x] I have tested my changes.
- [x] I have updated the documentation to reflect the changes.
- [x] I have run `task pyright` and fixed the relevant issues.

### CODE REVIEW

File: nextcord/channel.py [hunk 1]
```diff
@@ -1474,6 +1474,31 @@ async def create_webhook(
...
         return Webhook.from_state(data, state=self._state)
 
+    async def webhooks(self) -> List[Webhook]:
+        """|coro|
+
+        Gets the list of webhooks from this channel.
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
+
+        .. versionadded:: 2.6
```
Comment (EmreTech) [inline nextcord/channel.py:1484]:
```suggestion
        .. versionadded:: 3.0
```

File: nextcord/channel.py [hunk 2]
```diff
@@ -2014,6 +2039,31 @@ async def create_webhook(
...
```
Comment (EmreTech) [inline nextcord/channel.py:2049]:
```suggestion
        .. versionadded:: 3.0
```

File: nextcord/channel.py [hunk 3]
```diff
@@ -2330,6 +2380,31 @@ async def edit(self, *, reason=None, **options):
...
         return None
 
+    async def webhooks(self) -> List[Webhook]:
+        """|coro|
+
+        Gets the list of webhooks from this channel.
+
+        Requires :attr:`~.Permissions.manage_webhooks` permissions.
+
+        .. versionadded:: 2.6
```
Comment (EmreTech) [inline nextcord/channel.py:2390]:
```suggestion
        .. versionadded:: 3.0
```

**Review by EmreTech — CHANGES_REQUESTED** [review]
Looks good otherwise!

**Review by EmreTech — APPROVED** [review]

**Review by Skelmis — APPROVED** [review]

**Pull request merged**